tools/convert.rb

```diff
@@ -1128,17 +1128,23 @@ def parseUCIngest(itemID, inMeta, fileType, isPending)
 
   # For eschol journals, populate the issue and section models.
   issue = section = nil
-  volNum = inMeta.text_at("./context/volume")
-  issueNum = inMeta.text_at("./context/issue")
-  if inMeta[:state] != "withdrawn" && issueNum && volNum
+  if inMeta[:state] != "withdrawn"
+
+    # Figure out if the pub is from an eschol journal
     issueUnit = inMeta.xpath("./context/entity[@id]").select {
                       |ent| $allUnits[ent[:id]] && $allUnits[ent[:id]].type == "journal" }[0]
     issueUnit and issueUnit = issueUnit[:id]
+
     if issueUnit
-      # Data for eScholarship journals
-      if $allUnits.include?(issueUnit)
-        volNum.nil? and raise("missing volume number on eschol journal item")
```

Comment:
Is there a change in behavior for eschol journal item where issue or volume is absent?